src/new-bot/commands/expertise/lib/dialog.js

```diff
@@ -0,0 +1,102 @@
+import Promise from 'bluebird';
+import {normalizeResponse} from 'chatter';
+
+export function ask(options = {}) {
+  const {
+    header,
```

Comment:
I think it's going to be more common to want to pass a single `header` message instead of an array of `headers`, just like it's more common for a message handler to return a single `message` instead of an array of `messages`.